cumulusci/tasks/push/push_api.py

```diff
@@ -358,13 +373,22 @@ def get_package_versions_by_id(self, where=None, limit=None):
 
     @lru_cache(32)
     def get_subscribers(self, where=None, limit=None):
-        where = self.format_where_clause(where, obj="PackageSubscriber")
-        query = (
-            "SELECT Id, MetadataPackageVersionId, InstalledStatus, OrgName, OrgKey, OrgStatus, OrgType from PackageSubscriber%s"
-            % where
-        )
+        sObject = "PackageSubscriber"
+        where = self.format_where_clause(where, obj=sObject)
+        query = f"SELECT Id, MetadataPackageVersionId, InstalledStatus, OrgName, OrgKey, OrgStatus, OrgType from {sObject}{where}"
         query = self.add_query_limit(query, limit)
-        return self.return_query_records(query)
+        field_names = [
+            "Id",
+            "MetadataPackageVersionId",
+            "InstalledStatus",
+            "OrgName",
+            "OrgKey",
+            "OrgStatus",
+            "OrgType",
+        ]
```

Comment:
It would probably be a good idea to define this list once, then use it both for generating `query` and passing in as `field_names` -- to make sure that they don't accidentally get out of sync if we change one but forget to change the other.

Reply:
Personal reminder @jofsky to do this.